<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoDelS - Moral Deliberation Scenarios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>771</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_ego">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Ego</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_vehicles">
     <attribute name="title">
      <string>Vehicles</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_select_vehicle">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>6</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Vehicle</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>20</width>
        <height>171</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_vehicle_causal_factor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>261</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_driver_causal_factor">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>301</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_vehicle_causal_factor_description">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="midLineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>Vehicle causal factor description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_driver_causal_factor_description">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>361</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Driver causal factor description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_experience_factor">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Experience Factor</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_mental_or_emotional_factor">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Mental or Emotional Factor</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_phy_or_phy_factor">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Physical or Physiological Factor</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_pedestrian">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Pedestrian</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_sub_scenarios">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>771</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_generated_scenarios">
     <attribute name="title">
      <string>Sub Scenarios</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_related_scenarios">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>711</width>
        <height>151</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Run me</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Other Result</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_list_of_scenarios">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>List of Scenarios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_run_scenario">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update_scenario">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>600</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Update &amp; Run</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>600</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenario Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scenario_description">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>751</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenario description</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_scenario"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_about_models"/>
    <addaction name="action_licenses"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>Output</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOutput"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_exit_2"/>
   <addaction name="action_edit"/>
   <addaction name="action_save"/>
   <addaction name="action_print"/>
  </widget>
  <action name="action_open_scenario">
   <property name="text">
    <string>Open Scenario</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_about_models">
   <property name="text">
    <string>About MoDelS</string>
   </property>
  </action>
  <action name="action_licenses">
   <property name="text">
    <string>Licenses</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normalon>../../../Assets/ui/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset>
     <normalon>../../../Assets/ui/print.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="icon">
    <iconset>
     <normalon>../../../Assets/ui/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
  </action>
  <action name="action_exit_2">
   <property name="icon">
    <iconset>
     <normaloff>../../../Assets/ui/exit.png</normaloff>../../../Assets/ui/exit.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
